docs/advanced/default.yml.spec.md

```diff
@@ -252,11 +252,15 @@ splunk:
   * key::value pairs for environment variables that get written to ${SPLUNK_HOME}/etc/splunk-launch.conf
   * Default: null
 
-  secret: null
+  secret: <str>
```

Comment:
Figured I'd fix this to follow convention in this file, although unrelated to the ticket.